src/nvim/lua/executor.c

```diff
@@ -910,6 +897,27 @@ static int nlua_debug(lua_State *lstate)
   return 0;
 }
 
+const char *luaL_tolstring (lua_State *L, int idx, size_t *len) {
```

Comment:
`luaL_` isn't part of Nvim's namespace, so we shouldn't be defining functions with that prefix.

Reply:
What name is suitable? I am not sure Neovim will port some other luaL_ functions from Lua5.2.

Reply:
Based on the other code in this file, `nlua_` seems to be our prefix.

Reply:
OK, I will change it.